RetailCoder.VBE/Settings/HotkeySetting.cs

```diff
@@ -22,9 +23,15 @@ public class HotkeySetting
         public bool HasCtrlModifier { get; set; }
 
         [XmlIgnore]
-        public string Prompt
+        public string Prompt => RubberduckUI.ResourceManager.GetString("HotkeyDescription_" + Name, CultureInfo.CurrentUICulture);
+
+        public HotkeySetting()
         {
-            get { return RubberduckUI.ResourceManager.GetString("HotkeyDescription_" + Name, CultureInfo.CurrentUICulture); } 
+        }
+
+        public HotkeySetting(Type commandType)
```

Comment:
the type of command is utterly irrelevant to the hotkey. It just needs to know the command instance it's bound to. Pass a `CommandBase` instead. Or make it even cleaner by removing the i18n consideration from this class and separate the binding and execution from one another. 

---The settings--- nobody should use the `Name` property of `HotkeySetting` in the first place. After all we're attempting to bind hotkeys to **every command** and can accordingly use the existing command i18n structure.

This frees us from the circularity of the binding between Command and HotkeySetting. In the end the Hotkey should depend on the command, and the command shouldn't need to care about HotkeySettings